sync.py

```diff
@@ -23,30 +24,35 @@
 
 # Step 0. Settings to fetch from user or set globally for the sync.
 parser = argparse.ArgumentParser()
-parser.add_argument("environment", choices=["production", "staging"], help="which environment to sync over")
-parser.add_argument("-d", "--destination", help="the non-standard destination path on the remote host to sync to")
+parser.add_argument('environment', choices=['production', 'staging'], help='which environment to sync over')
+parser.add_argument('-d', '--destination', help='the non-standard destination path on the remote host to sync to')
+parser.add_argument('deployment', help='which deployment to sync over')
```

Comment:
Personally I try avoid having two positional arguments. Mainly because it requires me to remember in which order I should supply them. 😅 Suggestion: Add argument flags for both `environment` and `deployment` . If you decide to do this, remember to update the sync commands in the README.

Reply:
Done! I wasn't quite sure of how the deployment directory looks like in arteria's case, but that option is not required to sync.